corpus/autocmds.txt

```diff
@@ -3,6 +3,11 @@ Basic
 ================================================================================
 
 au BufNewFile *.html echo "foo"
+au! BufNewFile *.html echo "foo"
+autocmd BufNewFile *.html echo "foo"
+autocmd! BufNewFile *.html echo "foo"
+au!
+autocmd!
```

Comment:
Could you move those two in their own test in the file ?

Reply:
Done!